docs/examples/general/data_loading/webdataset.ipynb

```diff
@@ -0,0 +1,514 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #17.        return decode_augment(img, seed=seed), label
</pre><p>maybe personal preference. I think it'd read better if the whole pipeline was defined here (except for <code>read_webdataset</code>  function)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/3153/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
I thought that it might be better to define it separately since then we don't have to repeat it for the file_pipeline

Reply:
Sure